### PR TITLE
#3599 Improve whitelisting in issuer flow

### PR DESCRIPTION
**Pull request by drakkein**

### CODE REVIEW

File: packages/web/app/modules/bookbuilding-flow/utils.ts [hunk 1]
```diff
@@ -75,12 +76,14 @@ export const calculateWhitelistingState = ({
...
   isAuthorized,
+  isInvestor,
 }: TcalculateWhitelistingState) => {
   //TODO this should be put in line with api after API's canEnableBookbuilding after limit is reached gets fixed by Marcin
   if (
     isAuthorized &&
+    isInvestor &&
     whitelistingIsActive &&
-    (investorsCount === undefined || investmentCalculatedValues === undefined)
+    investmentCalculatedValues === undefined
```
Comment (Moejoe90) [inline packages/web/app/modules/bookbuilding-flow/utils.ts]: and what happens if hte user is not an investor?

Comment (drakkein) [reply packages/web/app/modules/bookbuilding-flow/utils.ts]: settled on slack

File: packages/web/intl/locales/en-en.json [hunk 2]
```diff
@@ -1116,12 +1116,13 @@
...
-  "settings.book-building-widget.start-book-building": "Enable subscription for your whitelist",
   "settings.book-building-widget.stop-book-building": "Disable whitelisting",
-  "settings.choose-pre-eto-date.book-building-will-stop": "Your ETO smart contracts have been deployed and your nominee signed all legal agreements. After setting the date of the ETO you will be asked to confirm all ETO terms and sign all legal agreements on-chain. Set the start date and time (UTC time) of your presale now. Public sale will automatically start after your presale ends. Unless you disable the subscription for the whitelist manually it will automatically close {minOffsetPeriod, plural, one {# day} other {# days}} before your chosen presale date",
+  "settings.book-building-widget.whitelist-description": "After Whitelisting, investors can sign up and indicate the amount they’d like to invest. You can disable the whitelist any time but please note - all of Neufund’s investors will be notified every time you disable and enable the whitelist. The whitelist will be automatically disabled 7 days before your presale starts.",
```
Comment (Moejoe90) [inline packages/web/intl/locales/en-en.json]: These shouldn't be hardcoded

File: packages/web/intl/locales/en-en.json [hunk 3]
```diff
@@ -1116,12 +1116,13 @@
...
+  "settings.choose-pre-eto-date.book-building-will-stop": "You have successfully set the date for your ETO. Make sure you communicate this to your community. The whitelist subscription will be automatically disabled 14 days before the start of the presale.",
```
Comment (Moejoe90) [inline packages/web/intl/locales/en-en.json]: This shouldn't be hardcoded at all

**Comment by Moejoe90** [comment]
> * Let me ask around about the days to close. We can set a presale to tomorrow, then why do we hardcode 7 days for the whitelisting? It shows 0.333 days on .io currently (also not so fortunate, but more accurate)

This shouldn't be hardcoded at all. this should be an automatic value coming from the backend so all answers will be correct for some cases and wrong for the other.

**Pull request merged**